docs/source/changelog.rst

```diff
@@ -14,7 +15,16 @@ Changelog
         * Update testing dependencies (:pr:`976`)
 
     Thanks to the following people for contributing to this release:
-    :user:`gsheni`, :user:`rwedge`
+    :user:`gsheni`, :user:`rwedge`, :user:`thehomebrewnerd`, :user:`sebrahimi1988`
+
+**Breaking Changes**
+
+* Calls to `featuretools.dfs` or `featuretools.calculate_feature_matrix` that use a cutoff time
```

Comment:
`featuretools.dfs`, etc., should be ``featuretools.dfs``

Reply:
Fixed